src/components/ListGroup.js

```diff
@@ -148,7 +148,10 @@ export default function ListGroup(props) {
           AddContainer
             ? (
               <AddContainer>
-                <button class="bio-properties-panel-group-header-button bio-properties-panel-add-entry">
+                <button
+                  title="Create new list item"
```

Comment:
This title is very generic. Are we okay with that or should we make this configurable (e.g. _Create new input_)?

Reply:
It's part of our [design prototype](https://fluffy-fiesta-212e1c7c.pages.github.io/prototypes/visual-ide/v15/), so it might be ok to keep it generic. We can update it afterward if we see a need though.

/cc @andreasgeier

Reply:
👍🏻